ports/stm32f4/mpconfigport.mk

```diff
@@ -7,16 +7,18 @@ MPY_TOOL_LONGINT_IMPL = -mlongint-impl=mpz
 INTERNAL_LIBM = 1
 
 # Chip supplied serial number, in bytes
-USB_SERIAL_NUMBER_LENGTH = 16
+USB_SERIAL_NUMBER_LENGTH = 24
 
 # Longints can be implemented as mpz, as longlong, or not
 LONGINT_IMPL = MPZ
 
 #Reduced feature set for early port
 CIRCUITPY_MINIMAL_BUILD = 1
 
-ifeq ($(MCU_SUB_VARIANT), stm32f412zx)
+#ifeq ($(MCU_SUB_VARIANT), stm32f412zx)
```

Comment:
`#` is comment marker in .mk files. You added `#` here and on endif. Not sure you meant that.

Reply:
This was intentional, since there's often desynchronized functionality between the two discovery boards caused by USB or other differences. So it regularly gets uncommented and used when adding new features. If you'd like that kind of reference comment committed for PRs let me know.

Reply:
I'd suggest deleting it and only adding it when needed.